src/components/CandidateContest.tsx

```diff
@@ -297,16 +245,25 @@ class CandidateContest extends React.Component<Props, State> {
   }
 
   public closeAttemptedVoteAlert = () => {
-    this.setState({ attemptedOvervoteCandidate: undefined })
+    // Delay to avoid passing tap to next screen
```

Comment:
this is gonna be painful to have everywhere and test for.

Reply:
Yes. Hoping to have a better solution based upon how much testing pain this causes.